2018-01-24 08:58:18.3574 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск бота 

2018-01-24 08:58:18.3964 | DeeBeeTeeAlphaBot.Program.Main | INFO | Подключение к БД 

2018-01-24 08:58:19.0056 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск пуллинга telegram API 

2018-01-24 08:59:22.6494 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 60 

2018-01-24 09:00:35.1583 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 120 

2018-01-24 09:01:45.3833 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 180 

2018-01-24 09:02:50.8821 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 240 

2018-01-24 09:03:56.2710 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 300 

2018-01-24 09:05:02.5052 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 360 

2018-01-24 09:06:12.3457 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 420 

2018-01-24 09:07:16.6708 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 480 

2018-01-24 09:08:21.2292 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 540 

2018-01-24 09:09:26.3188 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 600 

2018-01-24 09:10:33.9118 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 660 

2018-01-24 09:11:38.4664 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 720 

2018-01-24 09:12:42.8091 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 780 

2018-01-24 09:13:47.4207 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 840 

2018-01-24 09:14:52.6363 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 900 

2018-01-24 09:15:59.0959 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 960 

2018-01-24 09:17:05.4716 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1020 

2018-01-24 09:18:09.9093 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1080 

2018-01-24 09:19:14.6419 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1140 

2018-01-24 09:20:20.4085 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1200 

2018-01-24 09:21:38.2954 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1260 

2018-01-24 09:22:43.1566 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1320 

2018-01-24 09:23:48.5316 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1380 

2018-01-24 09:24:53.2044 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1440 

2018-01-24 09:26:03.5491 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1500 

2018-01-24 09:27:08.3165 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1560 

2018-01-24 09:28:13.1329 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1620 

2018-01-24 09:29:17.7964 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1680 

2018-01-24 09:30:23.6099 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1740 

2018-01-24 09:31:38.6207 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1800 

2018-01-24 09:32:47.7017 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1860 

2018-01-24 09:33:53.1830 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1920 

2018-01-24 09:34:57.5071 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1980 

2018-01-24 09:36:04.2192 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2040 

2018-01-24 09:37:12.1047 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2100 

2018-01-24 09:38:17.0505 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2160 

2018-01-24 09:39:21.8068 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2220 

2018-01-24 09:40:26.4319 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2280 

2018-01-24 09:41:41.9350 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2340 

2018-01-24 09:42:46.2666 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2400 

2018-01-24 09:43:53.2713 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2460 

2018-01-24 09:44:59.6246 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2520 

2018-01-24 09:46:11.2295 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2580 

2018-01-24 09:47:15.9268 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2640 

2018-01-24 09:48:21.2590 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2700 

2018-01-24 09:49:26.2957 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2760 

2018-01-24 09:50:30.8269 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2820 

2018-01-24 09:51:35.1300 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2880 

2018-01-24 09:52:39.5776 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2940 

2018-01-24 09:53:43.8505 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3000 

2018-01-24 09:54:48.4392 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3060 

2018-01-24 09:55:53.1143 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3120 

2018-01-24 09:57:03.9418 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3180 

2018-01-24 09:58:10.1265 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3240 

2018-01-24 09:59:14.8526 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3300 

2018-01-24 10:00:20.5039 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3360 

2018-01-24 10:08:46.5988 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 10:08:47.1169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 10:09:51.2227 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3420 

2018-01-24 10:10:57.2851 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3480 

2018-01-24 10:12:09.4828 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3540 

2018-01-24 10:13:17.2185 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3600 

2018-01-24 10:14:25.4086 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3660 

2018-01-24 10:15:36.0462 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3720 

2018-01-24 10:16:45.2399 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3780 

2018-01-24 10:17:53.4496 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3840 

2018-01-24 10:19:01.5503 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3900 

2018-01-24 10:20:06.5581 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3960 

2018-01-24 10:21:11.8158 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4020 

2018-01-24 10:22:16.3583 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4080 

2018-01-24 10:23:20.7374 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4140 

2018-01-24 10:24:25.3865 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4200 

2018-01-24 10:25:30.2197 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4260 

2018-01-24 10:26:34.7175 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4320 

2018-01-24 10:27:44.9794 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4380 

2018-01-24 10:28:51.2475 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4440 

2018-01-24 10:29:56.3135 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4500 

2018-01-24 10:31:05.1980 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4560 

2018-01-24 10:32:14.5753 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4620 

2018-01-24 10:33:22.0933 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4680 

2018-01-24 10:34:26.7628 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4740 

2018-01-24 10:35:34.2734 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4800 

2018-01-24 10:36:48.6647 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4860 

2018-01-24 10:37:57.5868 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4920 

2018-01-24 10:39:04.0365 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 4980 

2018-01-24 10:40:08.3198 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5040 

2018-01-24 10:41:15.1167 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5100 

2018-01-24 10:42:20.9318 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5160 

2018-01-24 10:43:35.4966 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5220 

2018-01-24 10:44:41.8441 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5280 

2018-01-24 10:45:46.1529 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5340 

2018-01-24 10:46:51.5002 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5400 

2018-01-24 10:47:59.2822 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5460 

2018-01-24 10:49:04.2658 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5520 

2018-01-24 10:50:09.1834 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5580 

2018-01-24 10:51:16.1598 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5640 

2018-01-24 10:52:26.1864 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5700 

2018-01-24 10:53:31.7199 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5760 

2018-01-24 10:54:37.3322 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5820 

2018-01-24 10:55:42.5750 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5880 

2018-01-24 10:56:49.4932 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 5940 

2018-01-24 10:57:59.5592 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6000 

2018-01-24 10:59:03.9395 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6060 

2018-01-24 11:00:08.7321 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6120 

2018-01-24 11:01:13.6975 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6180 

2018-01-24 11:02:18.0674 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6240 

2018-01-24 11:03:22.6204 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6300 

2018-01-24 11:04:28.6797 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6360 

2018-01-24 11:05:33.9810 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6420 

2018-01-24 11:06:43.8421 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6480 

2018-01-24 11:07:48.1273 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6540 

2018-01-24 11:08:53.4824 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6600 

2018-01-24 11:09:57.7977 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6660 

2018-01-24 11:11:03.1960 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6720 

2018-01-24 11:12:07.6427 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6780 

2018-01-24 11:13:11.9820 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6840 

2018-01-24 11:14:16.6113 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6900 

2018-01-24 11:15:20.9981 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 6960 

2018-01-24 11:16:28.6926 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7020 

2018-01-24 11:17:38.6304 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7080 

2018-01-24 11:18:43.4294 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7140 

2018-01-24 11:19:48.4351 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7200 

2018-01-24 11:20:53.4187 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7260 

2018-01-24 11:21:59.6358 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7320 

2018-01-24 11:23:04.2491 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7380 

2018-01-24 11:24:08.7546 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7440 

2018-01-24 11:30:24.5047 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.0433 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.0608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.1609 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.5151 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.6432 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.8456 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:25.9506 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.0120 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.0483 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.1100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.2551 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.2551 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4528 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4678 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4678 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4878 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.4998 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5158 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5158 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5158 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7500 

2018-01-24 11:30:26.5308 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5308 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5538 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5608 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5928 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5928 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.5928 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6239 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6239 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6239 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6239 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6469 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6469 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6589 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6589 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6779 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6779 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6929 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.6929 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7029 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7029 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7029 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7179 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7530 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7850 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7850 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7990 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.7990 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8100 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8290 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8290 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8290 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8420 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8420 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8420 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8420 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8740 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8740 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8740 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8740 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.8920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9041 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9041 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9041 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9041 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7560 

2018-01-24 11:30:26.9221 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9221 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9221 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9361 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9361 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9361 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9361 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9701 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9701 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9701 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:26.9831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0001 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0001 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0001 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0311 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0311 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0692 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0772 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0772 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0772 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0772 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.0952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1082 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1082 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1082 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1082 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1242 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1242 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1242 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1242 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1402 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1402 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1402 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1402 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1572 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1572 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1572 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1572 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.1862 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2033 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2033 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2033 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7620 

2018-01-24 11:30:27.2169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2362 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2362 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2362 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2492 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2492 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2492 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2632 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2632 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2632 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2632 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2822 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2822 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2822 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.2952 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3142 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3142 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3142 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3272 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3272 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3523 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3583 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3773 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3773 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.3908 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4038 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4038 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4273 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4273 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4413 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4413 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4538 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4538 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4538 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4673 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4853 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4853 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4853 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4983 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4983 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4983 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.4983 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5154 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5154 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5154 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5154 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5324 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5324 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5324 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5454 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7680 

2018-01-24 11:30:27.5454 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5454 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5604 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5604 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5604 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5604 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5794 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5794 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5794 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5934 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5934 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.5934 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6084 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6084 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6084 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6084 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6254 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6254 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6254 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6384 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6384 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6384 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6384 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6581 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6581 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6581 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6742 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6863 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6863 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6863 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.6863 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7039 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7336 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7336 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7336 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7336 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7792 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7792 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7792 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7792 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7961 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7961 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7961 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.7961 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8131 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8131 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7740 

2018-01-24 11:30:27.8131 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8256 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8256 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8256 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8256 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8436 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8436 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8436 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8436 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8606 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8606 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8606 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8726 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8726 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8726 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8726 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.8897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9197 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9197 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9197 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9197 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9367 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9667 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9857 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9857 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9857 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9997 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9997 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:27.9997 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0287 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0287 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0287 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0287 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0458 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0458 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0458 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0458 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0648 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0648 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0648 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0778 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7800 

2018-01-24 11:30:28.0778 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0948 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0948 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0948 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.0948 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1118 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1118 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1118 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1238 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1238 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1238 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1238 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1468 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1468 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:30:28.1618 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 11:31:21.4477 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7860 

2018-01-24 11:32:26.6451 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7920 

2018-01-24 11:33:38.0792 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 7980 

2018-01-24 11:34:43.1201 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8040 

2018-01-24 11:35:47.4641 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8100 

2018-01-24 11:36:52.0612 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8160 

2018-01-24 11:38:01.4153 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8220 

2018-01-24 11:39:09.4701 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8280 

2018-01-24 11:40:13.9013 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8340 

2018-01-24 11:41:20.7329 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8400 

2018-01-24 11:42:29.1298 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8460 

2018-01-24 11:43:53.9023 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8520 

2018-01-24 11:44:58.4566 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8580 

2018-01-24 11:46:03.3294 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8640 

2018-01-24 11:47:12.7866 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8700 

2018-01-24 11:48:23.7824 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8760 

2018-01-24 11:49:28.5001 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8820 

2018-01-24 11:50:33.8277 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8880 

2018-01-24 11:51:39.2442 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 8940 

2018-01-24 11:52:59.4436 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9000 

2018-01-24 11:54:05.6301 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9060 

2018-01-24 11:55:13.8399 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9120 

2018-01-24 11:56:21.1154 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9180 

2018-01-24 11:57:31.9134 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9240 

2018-01-24 11:58:41.6402 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9300 

2018-01-24 11:59:46.5393 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9360 

2018-01-24 12:21:59.5558 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:00.1360 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:00.3832 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:00.3962 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:00.4102 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9420 

2018-01-24 12:22:00.7729 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:00.9948 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.2152 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3308 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3729 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3729 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3869 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3869 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.3869 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4089 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4209 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4209 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4329 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4489 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4669 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4669 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4809 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4809 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4809 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.4989 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.5730 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.5730 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.5730 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.5950 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.5950 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6070 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6270 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6410 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6410 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6541 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6731 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6731 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6831 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6991 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6991 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.6991 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7141 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7301 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7301 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7451 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7631 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7771 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7771 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7771 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7771 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7941 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9480 

2018-01-24 12:22:01.7941 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.7941 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8072 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8072 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8462 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8542 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8542 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8542 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8542 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8722 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8722 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8722 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8852 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8852 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8852 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.8852 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9032 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9212 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9212 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9212 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9342 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9342 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9342 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9342 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9502 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9676 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9676 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9676 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9946 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9946 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9946 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:01.9946 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0116 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0116 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0116 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0116 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0286 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0286 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0286 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0416 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0416 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0416 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0416 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0586 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0586 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0586 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0586 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0756 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0756 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0756 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.0897 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9540 

2018-01-24 12:22:02.0897 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1057 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1307 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1347 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1347 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1347 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1557 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1557 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1764 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1849 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1849 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.1849 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2019 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2019 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2019 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2309 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2484 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2609 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2609 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2609 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2609 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2789 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2789 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2789 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2919 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2919 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2919 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.2919 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3225 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3225 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3225 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3225 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3870 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3870 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3870 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.3870 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4034 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4034 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9600 

2018-01-24 12:22:02.4034 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4164 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4344 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4344 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4344 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4504 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4504 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4504 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4644 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4644 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4644 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4644 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4815 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4815 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4815 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4815 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4985 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4985 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.4985 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5115 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5285 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5285 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5285 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5435 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5435 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5435 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5575 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5745 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5745 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5745 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5745 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5915 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5915 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.5915 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6035 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6215 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6215 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6215 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6215 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6392 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6523 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6523 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6523 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6523 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6689 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6689 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6689 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9660 

2018-01-24 12:22:02.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6819 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6989 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6989 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6989 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.6989 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7169 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7289 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7289 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7289 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7289 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7469 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7469 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7469 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7600 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7790 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7790 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7790 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.7920 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8090 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8270 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8270 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8270 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8400 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8580 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8580 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8580 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8705 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.8896 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9036 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9036 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9036 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9166 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9346 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9346 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9346 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9346 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9720 

2018-01-24 12:22:02.9516 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9516 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9516 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9646 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9646 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9646 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9646 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9816 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:02.9956 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0136 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0267 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0437 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0437 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0437 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0577 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0577 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0577 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0577 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0767 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0917 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0917 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.0917 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1067 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1223 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1223 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1223 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1369 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1369 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1369 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1514 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:03.1514 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 12:22:22.5860 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9780 

2018-01-24 12:23:28.0482 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9840 

2018-01-24 12:24:32.4668 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9900 

2018-01-24 12:25:36.7763 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 9960 

2018-01-24 12:26:41.2044 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10020 

2018-01-24 12:27:45.6807 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10080 

2018-01-24 12:28:50.1059 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10140 

2018-01-24 12:30:00.0259 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10200 

2018-01-24 12:31:05.5371 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10260 

2018-01-24 12:32:17.8101 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10320 

2018-01-24 12:33:29.2133 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10380 

2018-01-24 12:34:33.5663 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10440 

2018-01-24 12:35:38.6382 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10500 

2018-01-24 12:36:43.7593 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10560 

2018-01-24 12:37:54.6822 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10620 

2018-01-24 12:39:06.6535 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10680 

2018-01-24 12:40:11.4812 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10740 

2018-01-24 12:41:18.8125 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10800 

2018-01-24 12:42:40.0148 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10860 

2018-01-24 12:43:54.8669 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10920 

2018-01-24 12:45:01.5791 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 10980 

2018-01-24 12:46:09.4819 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11040 

2018-01-24 12:47:18.4116 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11100 

2018-01-24 12:48:31.9415 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11160 

2018-01-24 12:49:36.8808 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11220 

2018-01-24 12:50:43.3524 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11280 

2018-01-24 12:51:48.2087 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11340 

2018-01-24 12:53:07.1985 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11400 

2018-01-24 12:54:18.4776 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11460 

2018-01-24 12:55:26.7565 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11520 

2018-01-24 12:56:33.8144 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11580 

2018-01-24 12:57:40.4073 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11640 

2018-01-24 12:58:51.0875 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11700 

2018-01-24 12:59:57.6996 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11760 

2018-01-24 13:01:06.9529 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11820 

2018-01-24 13:02:15.5949 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11880 

2018-01-24 13:03:29.9002 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 11940 

2018-01-24 14:01:15.6186 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:15.7147 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:15.7537 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.1078 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.5511 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.6432 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.6562 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.7552 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.7683 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.8023 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.8333 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:16.8603 | Telegram.Request.TelegramRequest.GetUpdates | ERROR | Невозможно разрешить удаленное имя: 'api.telegram.org' 

2018-01-24 14:01:20.7030 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12000 

2018-01-24 14:02:28.6525 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12060 

2018-01-24 14:03:34.9634 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12120 

2018-01-24 14:04:45.6299 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12180 

2018-01-24 14:05:52.7036 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12240 

2018-01-24 14:06:59.3118 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12300 

2018-01-24 14:08:12.8286 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12360 

2018-01-24 14:09:28.2175 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12420 

2018-01-24 14:10:37.1389 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12480 

2018-01-24 14:11:41.8929 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12540 

2018-01-24 14:12:48.3462 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12600 

2018-01-24 14:14:01.9852 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12660 

2018-01-24 14:15:10.9067 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12720 

2018-01-24 14:16:20.1562 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12780 

2018-01-24 14:17:33.1319 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12840 

2018-01-24 14:18:46.0201 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12900 

2018-01-24 14:19:56.4024 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 12960 

2018-01-24 14:21:08.2994 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13020 

2018-01-24 14:22:16.0018 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13080 

2018-01-24 14:23:25.2762 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13140 

2018-01-24 14:24:37.4458 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13200 

2018-01-24 14:25:46.0235 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13260 

2018-01-24 14:26:54.3967 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13320 

2018-01-24 14:28:06.4870 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13380 

2018-01-24 14:29:22.3196 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13440 

2018-01-24 14:30:35.1157 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13500 

2018-01-24 14:31:40.9022 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13560 

2018-01-24 14:32:47.9079 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13620 

2018-01-24 14:33:52.4097 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13680 

2018-01-24 14:35:05.0098 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13740 

2018-01-24 14:36:10.8634 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13800 

2018-01-24 14:37:17.2066 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13860 

2018-01-24 14:38:22.4330 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13920 

2018-01-24 14:39:32.5414 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 13980 

2018-01-24 14:40:38.1518 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14040 

2018-01-24 14:41:43.0073 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14100 

2018-01-24 14:42:48.8687 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14160 

2018-01-24 14:44:08.4394 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14220 

2018-01-24 14:45:15.1274 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14280 

2018-01-24 14:46:30.3127 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14340 

2018-01-24 14:47:40.8978 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14400 

2018-01-24 14:48:50.2752 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14460 

2018-01-24 14:50:03.2291 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14520 

2018-01-24 14:51:10.8720 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14580 

2018-01-24 14:52:20.6690 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14640 

2018-01-24 14:53:28.6776 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14700 

2018-01-24 14:54:37.8120 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14760 

2018-01-24 14:55:42.6022 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14820 

2018-01-24 14:56:47.3196 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14880 

2018-01-24 14:57:52.7934 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 14940 

2018-01-24 14:58:58.9632 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15000 

2018-01-24 15:00:05.0252 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15060 

2018-01-24 15:01:10.9955 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15120 

2018-01-24 15:02:16.1258 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15180 

2018-01-24 15:03:22.9811 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15240 

2018-01-24 15:04:29.4538 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15300 

2018-01-24 15:05:36.3512 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15360 

2018-01-24 15:06:41.4824 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15420 

2018-01-24 15:07:45.9841 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15480 

2018-01-24 15:08:50.8383 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15540 

2018-01-24 15:10:01.8999 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15600 

2018-01-24 15:11:07.1839 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15660 

2018-01-24 15:12:12.8440 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15720 

2018-01-24 15:13:19.8466 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15780 

2018-01-24 15:14:26.9999 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15840 

2018-01-24 15:15:42.2087 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 15900 

2018-01-24 15:50:48.1127 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск бота 

2018-01-24 15:50:48.1497 | DeeBeeTeeAlphaBot.Program.Main | INFO | Подключение к БД 

2018-01-24 15:50:49.2116 | DeeBeeTeeAlphaBot.Program.Main | INFO | Запуск пуллинга telegram API 

2018-01-24 15:50:49.6199 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 15:50:49.6280 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:580 user_id:2730927 chat_id:-156558308 username:Infom name: Ruslan Kasimov date: 1516798233 message:'/t @Infom 992 @Infom @Sergeiaf @TretyakRoman' 

2018-01-24 15:50:50.0449 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-24 15:50:50.0589 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Infom 992 @Infom @Sergeiaf @TretyakRoman' 

2018-01-24 15:50:50.0679 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Infom' 

2018-01-24 15:50:50.0679 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '992' 

2018-01-24 15:50:50.0839 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Обнаружена мультитранзакция, поиск пользователей в подстроке '@Infom @Sergeiaf @TretyakRoman  

2018-01-24 15:50:50.0839 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Infom' 

2018-01-24 15:50:50.0959 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'Sergeiaf' 

2018-01-24 15:50:50.0959 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | add user  'TretyakRoman' 

2018-01-24 15:50:50.0959 | DeeBeeTeeDB.DBAPI.Command_transaction | TRACE | Количество пользователей to в мультитранзакции 3, сам пользователь True 

2018-01-24 15:50:50.1099 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 15:50:50.1099 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 15:50:50.1941 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '0' to 'Infom' 

2018-01-24 15:50:50.1941 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 15:50:50.2101 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 15:50:50.2920 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '1' to 'Sergeiaf' 

2018-01-24 15:50:50.2997 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 15:50:50.2997 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 15:50:50.3890 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь номер '2' to 'TretyakRoman' 

2018-01-24 15:50:50.3950 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-24 15:50:50.3950 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-24 15:50:50.4823 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 15:50:50.4903 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 15:50:50.5730 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-24 15:50:50.6513 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет -2613,00 

2018-01-24 15:50:50.6613 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Infom amount:330 to:Sergeiaf 

2018-01-24 15:50:50.6613 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Infom' ,330 ,'Sergeiaf' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-24 15:50:50.7826 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 15:50:50.8611 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-24 15:50:51.0852 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 15:50:51.0852 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 15:50:51.1752 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-24 15:50:51.2551 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Sergeiaf составляет 1275,00 

2018-01-24 15:50:51.2551 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Infom amount:332 to:TretyakRoman 

2018-01-24 15:50:51.2711 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Infom' ,332 ,'TretyakRoman' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-24 15:50:51.3703 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-24 15:50:51.4531 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-24 15:50:51.6574 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | ERROR | Удаленный сервер возвратил ошибку: (400) Недопустимый запрос. 

2018-01-24 15:50:51.6574 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-24 15:50:51.6734 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-24 15:50:51.7549 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'TretyakRoman'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'TretyakRoman'), 0) as Bal  

2018-01-24 15:50:51.8373 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @TretyakRoman составляет -213,00 

2018-01-24 15:50:51.8453 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Обнаружена команда мультитранзакции, будет добавлено несколько транзакция с делением суммы на целые части
Транзакция 85 успешно добавлена
Транзакция 86 успешно добавлена
Баланс пользователя @Infom составляет -2613,00
Баланс пользователя @Sergeiaf составляет 1275,00
Баланс пользователя @TretyakRoman составляет -213,00
 

2018-01-24 15:50:52.0212 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 15:50:52.0212 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 15:50:52.1120 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 15:50:52.1120 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 2730927) 

2018-01-24 15:51:57.0815 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 60 

2018-01-24 15:53:03.5082 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 120 

2018-01-24 16:01:51.1832 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:01:51.1872 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:584 user_id:459569 chat_id:-156558308 username:Sergeiaf name: Сергей Федоров date: 1516798877 message:'/t @Infom 1000 @Sergeiaf' 

2018-01-24 16:01:51.7713 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-24 16:01:51.7713 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Infom 1000 @Sergeiaf' 

2018-01-24 16:01:51.7803 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Infom' 

2018-01-24 16:01:51.7803 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1000' 

2018-01-24 16:01:51.7803 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'Sergeiaf' 

2018-01-24 16:01:51.7933 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:01:51.7933 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:01:51.8924 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:01:51.8924 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:51.9744 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Infom amount:1000 to:Sergeiaf 

2018-01-24 16:01:51.9744 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Infom' ,1000 ,'Sergeiaf' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-24 16:01:52.0679 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:52.1470 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-24 16:01:52.3113 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:01:52.3113 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:01:52.3933 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-24 16:01:52.4854 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет -951,00 

2018-01-24 16:01:52.4854 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:01:52.4854 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:52.5734 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-24 16:01:52.6525 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Sergeiaf составляет 275,00 

2018-01-24 16:01:52.6525 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 87 успешно добавлена
Баланс пользователя @Infom составляет -951,00
Баланс пользователя @Sergeiaf составляет 275,00 

2018-01-24 16:01:52.8396 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:01:52.8396 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Sergeiaf' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 16:01:52.9227 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:01:52.9227 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 459569, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 459569) 

2018-01-24 16:01:53.2612 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:01:53.2682 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:585 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516798892 message:'/t @infom 1000 @Sergeiaf' 

2018-01-24 16:01:53.2682 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-24 16:01:53.2795 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@infom 1000 @Sergeiaf' 

2018-01-24 16:01:53.2795 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'infom' 

2018-01-24 16:01:53.2795 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1000' 

2018-01-24 16:01:53.2941 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'Sergeiaf' 

2018-01-24 16:01:53.2941 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'infom' 

2018-01-24 16:01:53.2941 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-24 16:01:53.3833 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:01:53.3883 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:53.4672 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:infom amount:1000 to:Sergeiaf 

2018-01-24 16:01:53.4672 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('infom' ,1000 ,'Sergeiaf' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-24 16:01:53.5633 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:53.6403 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-24 16:01:53.8475 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'infom' 

2018-01-24 16:01:53.8475 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'infom' 

2018-01-24 16:01:53.9295 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'infom'), 0) as Bal  

2018-01-24 16:01:54.0106 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @infom составляет 49,00 

2018-01-24 16:01:54.0156 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:01:54.0156 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:01:54.1026 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-24 16:01:54.2027 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Sergeiaf составляет -725,00 

2018-01-24 16:01:54.2027 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 88 успешно добавлена
Баланс пользователя @infom составляет 49,00
Баланс пользователя @Sergeiaf составляет -725,00 

2018-01-24 16:01:54.4170 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:01:54.4220 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-24 16:01:54.5275 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:01:54.5275 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-24 16:02:58.4609 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 180 

2018-01-24 16:03:08.2441 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:03:08.2491 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:592 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516798987 message:'/d' 

2018-01-24 16:03:08.2491 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-24 16:03:08.2491 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:03:08.2641 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:03:08.3531 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-24 16:03:08.4352 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Infom' GROUP BY to_user 

2018-01-24 16:03:08.5143 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: amarshalkin amount: -128,00  

2018-01-24 16:03:08.5143 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Kasim_RI amount: -761,00  

2018-01-24 16:03:08.5143 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Sergeiaf amount: 881,00  

2018-01-24 16:03:08.5273 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Svetlana_Shulgina amount: 0,00  

2018-01-24 16:03:08.5273 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: TretyakRoman amount: 57,00  

2018-01-24 16:03:08.5273 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Infom. Баланс: 49,00
 @Infom должен 128,00 @amarshalkin
 @Infom должен 761,00 @Kasim_RI
 @Sergeiaf должен 881,00 @Infom
 @TretyakRoman должен 57,00 @Infom
 

2018-01-24 16:03:08.7375 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:03:08.7375 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-24 16:03:08.8194 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:03:08.8244 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-24 16:04:04.7789 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 240 

2018-01-24 16:05:10.8195 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 300 

2018-01-24 16:05:41.1785 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:05:41.1875 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:595 user_id:459569 chat_id:-156558308 username:Sergeiaf name: Сергей Федоров date: 1516799140 message:'/d@DeeBeeTeeBot' 

2018-01-24 16:05:41.1875 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-24 16:05:41.2065 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:05:41.2065 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:05:41.2919 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-24 16:05:41.3700 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Sergeiaf' GROUP BY to_user 

2018-01-24 16:05:41.4562 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Sergeiaf to: Infom amount: -881,00  

2018-01-24 16:05:41.4562 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Sergeiaf to: TretyakRoman amount: 156,00  

2018-01-24 16:05:41.4722 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Sergeiaf. Баланс: -725,00
 @Sergeiaf должен 881,00 @Infom
 @TretyakRoman должен 156,00 @Sergeiaf
 

2018-01-24 16:05:41.6428 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:05:41.6428 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Sergeiaf' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 16:05:41.7507 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:05:41.7507 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 459569, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 459569) 

2018-01-24 16:06:15.8107 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 360 

2018-01-24 16:06:25.5811 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:06:25.5811 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:598 user_id:459569 chat_id:-156558308 username:Sergeiaf name: Сергей Федоров date: 1516799184 message:'/t @Sergeiaf 1000 @Infom' 

2018-01-24 16:06:25.5977 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /t 

2018-01-24 16:06:25.5977 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Начала парсинга транзакции в строке '@Sergeiaf 1000 @Infom' 

2018-01-24 16:06:25.6117 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь from 'Sergeiaf' 

2018-01-24 16:06:25.6117 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Сумма '1000' 

2018-01-24 16:06:25.6217 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Пользователь to 'Infom' 

2018-01-24 16:06:25.6217 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:06:25.6217 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:06:25.7108 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:06:25.7158 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:06:25.7937 | DeeBeeTeeDB.DBAPI.Command_transaction | DEBUG | Выполнение в SQL транзакции from:Sergeiaf amount:1000 to:Infom 

2018-01-24 16:06:25.7937 | DeeBeeTeeDB.DBAPI.NewTransaction | TRACE | NewTransactionSQLParam: INSERT INTO [dbo].[transactions] ([from_user] ,[amount] ,[to_user] ,[operation_date] ,[oid]) VALUES ('Sergeiaf' ,1000 ,'Infom' ,getdate() ,0) SELECT top 1 tid FROM [dbo].[transactions] order by [operation_date] DESC 

2018-01-24 16:06:25.8852 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:06:25.9678 | DeeBeeTeeAlphaBot.Program.RegisterTransaction | INFO | Отправка сообщения пользователю о новой транзакции в личный чат 

2018-01-24 16:06:26.1431 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Sergeiaf' 

2018-01-24 16:06:26.1521 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Sergeiaf' 

2018-01-24 16:06:26.2341 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Sergeiaf'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Sergeiaf'), 0) as Bal  

2018-01-24 16:06:26.3161 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Sergeiaf составляет 275,00 

2018-01-24 16:06:26.3161 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:06:26.3274 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:06:26.4062 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-24 16:06:26.4871 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @Infom составляет -951,00 

2018-01-24 16:06:26.4871 | DeeBeeTeeDB.DBAPI.Command_transaction | INFO | Возврат результата команды transaction с сообщением Транзакция 89 успешно добавлена
Баланс пользователя @Sergeiaf составляет 275,00
Баланс пользователя @Infom составляет -951,00 

2018-01-24 16:06:26.6590 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:06:26.6680 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'Sergeiaf' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 16:06:26.7511 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:06:26.7511 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 459569, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 459569) 

2018-01-24 16:06:42.5510 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:06:42.5510 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:602 user_id:2730927 chat_id:2730927 username:Infom name: Ruslan Kasimov date: 1516799201 message:'/d' 

2018-01-24 16:06:42.5610 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-24 16:06:42.5610 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'Infom' 

2018-01-24 16:06:42.5610 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'Infom' 

2018-01-24 16:06:42.6484 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'Infom'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'Infom'), 0) as Bal  

2018-01-24 16:06:42.7305 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'Infom' GROUP BY to_user 

2018-01-24 16:06:42.8109 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: amarshalkin amount: -128,00  

2018-01-24 16:06:42.8109 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Kasim_RI amount: -761,00  

2018-01-24 16:06:42.8109 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Sergeiaf amount: -119,00  

2018-01-24 16:06:42.8109 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: Svetlana_Shulgina amount: 0,00  

2018-01-24 16:06:42.8269 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: Infom to: TretyakRoman amount: 57,00  

2018-01-24 16:06:42.8269 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @Infom. Баланс: -951,00
 @Infom должен 128,00 @amarshalkin
 @Infom должен 761,00 @Kasim_RI
 @Infom должен 119,00 @Sergeiaf
 @TretyakRoman должен 57,00 @Infom
 

2018-01-24 16:06:43.0030 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:06:43.0030 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT 2730927 ,'private' ,'', 'Infom' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = 2730927) 

2018-01-24 16:06:43.0931 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:06:43.0931 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT 2730927 , 2730927, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = 2730927 AND [user_id] = 2730927) 

2018-01-24 16:07:00.5774 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:07:00.5774 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:605 user_id:288805897 chat_id:-156558308 username:TretyakRoman name: Третьяк Роман date: 1516799220 message:'/d' 

2018-01-24 16:07:00.5774 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /d 

2018-01-24 16:07:00.5904 | DeeBeeTeeDB.DBAPI.Command_details | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-24 16:07:00.5904 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-24 16:07:00.6725 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'TretyakRoman'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'TretyakRoman'), 0) as Bal  

2018-01-24 16:07:00.7548 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | GetUserDetailsSQL: SELECT to_user, SUM(amount) as amount FROM (SELECT [tid], [from_user], [amount], [to_user] FROM [dbo].[transactions] union SELECT [tid], [to_user], [amount]*(-1), [from_user] FROM [dbo].[transactions] ) AS T WHERE from_user = 'TretyakRoman' GROUP BY to_user 

2018-01-24 16:07:00.8337 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: TretyakRoman to: Infom amount: -57,00  

2018-01-24 16:07:00.8337 | DeeBeeTeeDB.DBAPI.GetUserDetails | TRACE | Generate dbt from: TretyakRoman to: Sergeiaf amount: -156,00  

2018-01-24 16:07:00.8427 | DeeBeeTeeDB.DBAPI.Command_details | INFO | Возврат результата команды details с сообщениемДетали пользователя @TretyakRoman. Баланс: -213,00
 @TretyakRoman должен 57,00 @Infom
 @TretyakRoman должен 156,00 @Sergeiaf
 

2018-01-24 16:07:01.0286 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:07:01.0286 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'TretyakRoman' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 16:07:01.1117 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:07:01.1117 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 288805897, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 288805897) 

2018-01-24 16:07:20.8244 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 420 

2018-01-24 16:07:23.0450 | Telegram.Request.TelegramRequest.GetUpdates | DEBUG | New message event 

2018-01-24 16:07:23.0450 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | DEBUG | New message: message_id:608 user_id:288805897 chat_id:-156558308 username:TretyakRoman name: Третьяк Роман date: 1516799242 message:'/b' 

2018-01-24 16:07:23.0450 | DeeBeeTeeAlphaBot.Program.Tr_MessageText | INFO | Получение команды /b 

2018-01-24 16:07:23.0580 | DeeBeeTeeDB.DBAPI.Command_balance | DEBUG | Поиск пользователя 'TretyakRoman' 

2018-01-24 16:07:23.0580 | DeeBeeTeeDB.DBAPI.SearchUser | TRACE | UserSearchSQL: SELECT uid, [user], assign_date, limit, user_id from dbo.[users] where [user] = 'TretyakRoman' 

2018-01-24 16:07:23.1392 | DeeBeeTeeDB.DBAPI.GetUserBalance | TRACE | GetUserBalanceSQL: SELECT ISNULL((select SUM(amount) from dbo.[transactions] WHERE [from_user] = 'TretyakRoman'),0) - ISNULL((select SUM(amount) from dbo.[transactions] WHERE [to_user] = 'TretyakRoman'), 0) as Bal  

2018-01-24 16:07:23.2747 | DeeBeeTeeDB.DBAPI.Command_balance | INFO | Возврат результата команды balance с сообщением Баланс пользователя @TretyakRoman составляет -213,00 

2018-01-24 16:07:23.4567 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat update 

2018-01-24 16:07:23.4567 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUpdateSQL: INSERT INTO [dbo].[chats] ([chat_id],[type],[title],[username]) SELECT -156558308 ,'group' ,'ОФФшор', 'TretyakRoman' WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chats] WHERE [chat_id] = -156558308) 

2018-01-24 16:07:23.5395 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | Chat users update 

2018-01-24 16:07:23.5435 | DeeBeeTeeDB.DBAPI.UpdateChat | TRACE | ChatUsersUpdateSQL: INSERT INTO [dbo].[chatusers] ([chat_id],[user_id],[date_reg]) SELECT -156558308 , 288805897, getdate() WHERE NOT EXISTS (SELECT NULL FROM [dbo].[chatusers] WHERE [chat_id] = -156558308 AND [user_id] = 288805897) 

2018-01-24 16:08:25.0380 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 480 

2018-01-24 16:09:31.5920 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 540 

2018-01-24 16:10:37.1346 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 600 

2018-01-24 16:11:43.1287 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 660 

2018-01-24 16:12:48.6432 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 720 

2018-01-24 16:13:54.9967 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 780 

2018-01-24 16:15:01.1749 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 840 

2018-01-24 16:16:05.7771 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 900 

2018-01-24 16:17:11.6966 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 960 

2018-01-24 16:18:16.2908 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1020 

2018-01-24 16:19:21.7792 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1080 

2018-01-24 16:20:28.5711 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1140 

2018-01-24 16:21:36.3998 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1200 

2018-01-24 16:22:40.9180 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1260 

2018-01-24 16:23:47.7138 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1320 

2018-01-24 16:24:55.1085 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1380 

2018-01-24 16:26:02.4830 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1440 

2018-01-24 16:27:08.5343 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1500 

2018-01-24 16:28:14.8350 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1560 

2018-01-24 16:29:21.3846 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1620 

2018-01-24 16:30:27.8485 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1680 

2018-01-24 16:31:36.8504 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1740 

2018-01-24 16:32:42.5304 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1800 

2018-01-24 16:33:47.4939 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1860 

2018-01-24 16:34:53.4504 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1920 

2018-01-24 16:35:58.4994 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 1980 

2018-01-24 16:37:03.2550 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2040 

2018-01-24 16:38:07.6819 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2100 

2018-01-24 16:39:17.2646 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2160 

2018-01-24 16:40:24.2144 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2220 

2018-01-24 16:41:36.2512 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2280 

2018-01-24 16:42:44.4192 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2340 

2018-01-24 16:43:53.4502 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2400 

2018-01-24 16:44:59.0299 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2460 

2018-01-24 16:46:07.2549 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2520 

2018-01-24 16:47:12.5266 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2580 

2018-01-24 16:48:17.3369 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2640 

2018-01-24 16:49:21.9555 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2700 

2018-01-24 16:50:27.3024 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2760 

2018-01-24 16:51:34.4885 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2820 

2018-01-24 16:52:39.8951 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2880 

2018-01-24 16:53:45.9331 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 2940 

2018-01-24 16:54:52.0482 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3000 

2018-01-24 16:55:59.6066 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3060 

2018-01-24 16:57:08.0357 | Telegram.Request.TelegramRequest.GetUpdates | TRACE | HTTP request number 3120 

